lib/commons/aria/index.js

```diff
@@ -2387,43 +2387,54 @@ lookupTable.evaluateRoleForElement = {
 };
 
 /**
- * Reference -> https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#Widget_roles
  * The current lookupTable.role['widget'] widget, yeilds
  * ->
  * [
  * 	"alert", "alertdialog", "button", "checkbox", "dialog", "gridcell", "link", "log", "marquee", "menuitem", "menuitemcheckbox",
  * 	"menuitemradio", "option", "progressbar", "radio", "scrollbar", "searchbox", "slider", "spinbutton", "status", "switch", "tab", "tabpanel",
  * 	"textbox", "timer", "tooltip", "treeitem"
  * ]
- * There are some differences against specs, hence the below listing was made
+ * There are some differences against specs, hence the below listing was made by cross referring with `aria-query`
  */
 lookupTable.rolesOfType = {
 	widget: [
+		'command',
```

Comment:
command is an abstract role, not a widget role. Same for a lot of the ones you've added. Axe doesn't allow abstract roles on elements so we don't need to add them here.

Also don't we need to change `tooltip` in the a `lookupTable.role` to not be a widget role? I'm not sure how we differentiate between `widget` roles and `rolesOfType.widget` (such as `alertdialog` being a widget in the lookupTable but not in this list)

Reply:
Thanks @straker. 

The reason I did not use the above code snippet is our lookupTable is out of date & a small change to accommodate something like this usually breaks other tests. 

We have had conversations to update the lookupTable in the past. I used aria-query to generate this list of roles with a widget role type.

Let me discuss with @WilcoFiers for the right way forward to be lookupTable.